oat-sa/extension-tao-itemqti
# Feature/ms 67/external scoring

Tickets:

FE: https://oat-sa.atlassian.net/browse/MS-67
BE: https://oat-sa.atlassian.net/browse/MS-56

How to test?

- Install TAO
- Create an item 
- Click on the Response tab of the interaction and add a new outcome declaration from the right panel.
- Set the External Scored attribute to `human`
- Now when you change the min-max values of the outcome, you should get a warning on invalid scoring trait values (decimals) but it won't stop you from creating an item.
- After you save the item, you should be able to add it to a test, be able to deliver a test and have everything working as usual.
- you should also be able to export the test and have externalScored attribute on item's `OutcomeDeclaration` in XML.
